src/lib/s3Uploader.js

```diff
@@ -1,12 +1,26 @@
 import { S3 } from 'aws-sdk';
 
-export const s3 = new S3({
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-  endpoint: process.env.S3_ENDPOINT,
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
-  signatureVersion: 'v4',
-  s3ForcePathStyle: true,
-});
+let s3Config;
+
+if (process.env.VCAP_SERVICES) {
+  const bucket = process.env.VCAP_SERVICES.s3[0];
+  s3Config = {
+    accessKeyId: bucket.access_key_id,
+    endpoint: bucket.uri,
+    secretAccessKey: bucket.secret_access_key,
```

Comment:
I think these paths won't quite be correct. See the example data I DM'd you in slack. There's a `credentials` object that they're held in inside of what you've assigned to `bucket`